prometheus-metrics-core/src/test/java/io/prometheus/metrics/core/metrics/SlidingWindowTest.java

```diff
@@ -40,7 +40,7 @@ void assertValues(double... expectedValues) {
   private SlidingWindow<Observer> ringBuffer;
   private final long maxAgeSeconds = 30;
   private final int ageBuckets = 5;
-  private final long timeBetweenRotateMillis = maxAgeSeconds * 1000 / ageBuckets + 2;
+  private final long timeBetweenRotateMillis = maxAgeSeconds * 1000 / ageBuckets;
```

Comment:
This fix assumes that dropping the `+ 2` doesn't reduce coverage in a meaningful way. At first glance that appears to be so :eyes:

Reply:
I reverted this change; the updated fix only takes away the source of non-determinism.